arch/arm/src/c5471/c5471_ethernet.c

```diff
@@ -1338,21 +1338,6 @@ static void c5471_receive(struct c5471_driver_s *priv)
           if (dev->d_len > 0 &&
              (EIM_TXDESC_OWN_HOST & getreg32(priv->c_rxcpudesc)) == 0)
             {
-              /* Update the Ethernet header with the correct MAC address */
```

Comment:
let's move arp_ipin  at line 1320 into to ipv4_input

Reply:
Done